service-mango-fills/src/main.rs

```diff
@@ -58,6 +59,7 @@ pub struct Peer {
     pub head_updates: bool,
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
You did not add any arguments why did you add this allow clippy.
Could be another commit to fix clippy messages.

Reply:
true, I will isolate clippy fixes to a new PR

Reply:
done